jpa/eclipselink.jpa.test.jse/src/it/java/org/eclipse/persistence/jpa/test/query/TestSimpleQuery.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2021 IBM Corporation, Oracle, and/or affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+// Contributors:
+//     IBM - Bug 521402: Add support for Criteria queries with only literals
+package org.eclipse.persistence.jpa.test.query;
+
+import java.util.List;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.EntityManagerFactory;
+import jakarta.persistence.Query;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.ParameterExpression;
+import jakarta.persistence.criteria.Root;
+
+import org.eclipse.persistence.config.HintValues;
+import org.eclipse.persistence.config.QueryHints;
+import org.eclipse.persistence.jpa.test.framework.DDLGen;
+import org.eclipse.persistence.jpa.test.framework.Emf;
+import org.eclipse.persistence.jpa.test.framework.EmfRunner;
+import org.eclipse.persistence.jpa.test.framework.Property;
+import org.eclipse.persistence.jpa.test.framework.SQLCallListener;
+import org.eclipse.persistence.jpa.test.query.model.SimpleQueryEntity;
+import org.eclipse.persistence.jpa.test.query.model.SimpleQueryEntity_;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(EmfRunner.class)
+public class TestSimpleQuery {
+    @Emf(name = "SimpleQueryPersistenceUnit", 
+            createTables = DDLGen.DROP_CREATE, 
+            classes = { SimpleQueryEntity.class }, 
+            properties = { 
+                    @Property(name = "eclipselink.logging.level", value = "ALL"),
+                    @Property(name = "eclipselink.logging.level.sql", value = "FINE"),
```

Comment:
can this logging be configurable from the command line somehow?

Reply:
I have no idea how to hook command line arguments into the JSE bucket test runner. I can remove these logging properties if you are concerned with them. They should not impact the test and are really just there in case there is a failure in the future.

Reply:
`EmfRunnerInjector.inject` could add provided logging related system properties to PU... I have two use-cases:
* travis kills the job after certain amount of output, so in this case we probably want to completely disable logging
* on local box I want to easily increase the log level in case of failures but I don't want to see all details by default (ie info is sufficient in this case)